Library/Formula/cutecom.rb

```diff
@@ -0,0 +1,18 @@
+class Cutecom < Formula
+  homepage "http://cutecom.sourceforge.net"
+  url "http://cutecom.sourceforge.net/cutecom-0.22.0.tar.gz"
+  version "0.22.0"
+  sha1 "a23f9ddf3d1a4467568c872477fc8c6da07c1711"
+
+  depends_on "cmake" => :build
+  depends_on "qt" => ["with-qt3support"]
```

Comment:
I think this is a non-starter; we can't really include stuff that depends on Qt with qt3support as it will take a very long time to build.